packages/components/src/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,127 @@
+import type { StoryObj } from "@storybook/react";
+import React from "react";
+import EDSCheckbox, { CheckboxInput, Label } from "./Checkbox";
+
+type Args = React.ComponentProps<typeof EDSCheckbox>;
+
+/**
+ * Controlled example to make checked stories interactive.
+ * We name this "Checkbox" for Storybook's code documentation
+ */
+function Checkbox({ checked: defaultChecked = false, ...rest }: Args) {
+  const [checked, setChecked] = React.useState(defaultChecked);
+  const handleChange = () => {
+    setChecked(!checked);
+  };
+
+  return <EDSCheckbox checked={checked} onChange={handleChange} {...rest} />;
+}
+
+export default {
+  title: "Checkbox",
+  component: Checkbox,
+  args: {
+    disabled: false,
+    label: "Checkbox",
+  },
+  argTypes: {
+    // For some reason Storybook does not infer all props correctly;
+    // we manually include the most relevant controls here.
+    checked: {
+      description: `Whether checkbox is checked. Defaults to false.
+        "indeterminate" can be used when a checkbox visually represents
+        a list of checkboxes that is "partially" checked.`,
+      control: "radio",
+      options: [true, false, "indeterminate"],
```

Comment:
The `checked` controls aren't doing anything for me in the storybook. They do not update the state of the story. I think the story component may need to be updated because the checked prop is only a `defaultChecked` prop.

https://61313967cde49b003ae2a860-tshkmlvpue.chromatic.com/?path=/story/checkbox--default

Reply:
oh no, weird, it works locally 😭 will investigate

Reply:
gah something weird is happening with the state assignment:
<img width="800" alt="Screen Shot 2022-01-18 at 1 24 18 PM" src="https://user-images.githubusercontent.com/15840841/149996337-0732ae9c-50ed-4b6f-90c6-f434649eb4b2.png">

I'm gonna timebox this and just go back to the traject structure of stories